libs/designsystem/src/lib/components/page/page.component.ts

```diff
@@ -256,6 +276,15 @@ export class PageComponent
 
   ngOnInit(): void {
     this.removeWrapper();
+    this.refresherActive$
+      .pipe(
+        takeUntil(this.ngOnDestroy$),
+        filter((value) => value === true || value === false),
+        distinctUntilChanged()
+      )
```

Comment:
takeUntil should be the last - see https://ncjamieson.com/avoiding-takeuntil-leaks/